app/src/test/java/com/kickstarter/viewmodels/CrowdfundCheckoutViewModelTest.kt

```diff
@@ -788,7 +792,10 @@ class CrowdfundCheckoutViewModelTest : KSRobolectricTestCase() {
             viewModel.provideScopeAndDispatcher(this, dispatcher)
             viewModel.provideBundle(bundle)
             viewModel.userChangedPaymentMethodSelected(cards.last())
-            viewModel.pledgeOrUpdatePledge()
+            viewModel.pledgeOrUpdatePledge(
+                selectedCard = null,
+                isIncremental = true
```

Comment:
@jlplks similar question, if the test is `plot is not allowed on project` ? should `isIncremental` be true ?